src/controllers/private/v1/consent.private.controller.ts

```diff
@@ -39,6 +39,30 @@ export const getMyConsent = async (
     }
 };
 
+/**
+ * Get consent by user JWT from consent manager
```

Comment:
This doesn't seem to be the right documentation for the endpoint.
Also, documentating params in the context of middlewares don't seem to be really needed as it will always be req, res, next